src/source/tcp.rs

```diff
@@ -108,36 +124,30 @@ impl Source for Int {
                     // TODO also add token_num here?
                     path: path.clone(), // captures server port
                 };
+                let tls_acceptor: Option<TlsAcceptor> = conf
+                    .tls
+                    .as_ref()
+                    .and_then(|tls_config| load_server_config(tls_config).ok())
```

Comment:
we should fail when the certificates can't be loaded not silently switch to non SSL mode. That could be quite confusing for operators and be quite a security issue.